user/transaction/repository/gorm_transction_repository.go

```diff
@@ -35,11 +36,20 @@ func (tr *mysqlTransactionRepository) CreateTransaction(transaction entity.Trans
 	return transaction, nil
 }
 
-func (tr *mysqlTransactionRepository) GetAll(userId, trStatus, search string) ([]entity.Transaction, error) {
+func (tr *mysqlTransactionRepository) GetAll(userId, search string, trStatus... string) ([]entity.Transaction, error) {
 	allUserTransaction := []entity.Transaction{}
-	gormQuery := tr.DB.
-		Preload("Counselor").
-		Where("user_id = ? AND status = ?", userId, trStatus)
+	gormQuery := tr.DB.Debug().Preload("Counselor")
+
+	log.Println(trStatus[0])
+	
+	if len(trStatus) > 1 {
+		gormQuery.
+			Where("user_id = ?", userId).
+			Where("status = ? OR status = ?", trStatus[0], trStatus[1])
+	} else {
+		gormQuery.
+			Where("user_id = ? AND status = ?", userId, trStatus[0])
```

Comment:
ini gk bisa querynya di jadiin satu, jadi "gormQuery.
	Where("user_id = ?", userId).
	Where("status IN (?)", trStatus)". dicoba tu boleh

Reply:
ooh okee